src/status_im2/contexts/quo_preview/switcher/group_messaging_card.cljs

```diff
@@ -0,0 +1,125 @@
+   {:label "Counter Label"
+    :key   :counter-label
+    :type  :text}
+   {:label   "Type"
+    :key     :type
+    :type    :select
+    :options [{:key   :message
+               :value :text}
+              {:key   :photo
+               :value :photo}
+              {:key   :sticker
+               :value :sticker}
+              {:key   :gif
+               :value :gif}
+              {:key   :audio
+               :value :audio}
+              {:key   :community
+               :value :community}
+              {:key   :link
+               :value :link}
+              {:key   :code
+               :value :code-snippet}]}
+   {:label "Last Message"
+    :key   :last-message
+    :type  :text}
+   (preview/customization-color-option)])
+
+;; Mock data
+(def sticker {:source (resources/get-mock-image :sticker)})
+(def community-avatar {:source (resources/get-mock-image :community-logo)})
+(def gif {:source (resources/get-mock-image :gif)})
+(def coinbase-community (resources/get-mock-image :coinbase))
+(def photos-list
+  [{:source (resources/get-mock-image :photo1)}
+   {:source (resources/get-mock-image :photo2)}
+   {:source (resources/get-mock-image :photo3)}
+   {:source (resources/get-mock-image :photo1)}
+   {:source (resources/get-mock-image :photo2)}
+   {:source (resources/get-mock-image :photo3)}])
+
+(defn get-mock-content
+  [data]
+  (case (:type data)
+    :message
+    {:text (:last-message data)}
+
+    :photo
+    {:photos photos-list}
+
+    :sticker
+    sticker
+
+    :gif
+    gif
+
+    :audio
+    {:duration "00:32"}
+
+    :community
+    {:community-avatar coinbase-community
+     :community-name   "Coinbase"}
+
+    :link
+    {:source (resources/get-mock-image :status-logo)
+     :text   "Rolling St..."}
+
+    :code
+    nil))
+
+(defn get-mock-data
+  [data]
+  (merge
+   data
+   {:content (merge (get-mock-content data)
+                    {:mention-count (when (= (:status data) :mention) (:counter-label data))})}))
+
+(defn cool-preview
+  []
+  (let [state (reagent/atom {:title               "Hester, John, Steven, and 2 others"
+                             :type                :message
+                             :status              :read
+                             :last-message        "Hello there, there is a new message"
+                             :customization-color :camel
+                             :avatar              true
+                             :counter-label       5})]
+    (fn []
+      [rn/touchable-without-feedback {:on-press rn/dismiss-keyboard!}
+       [rn/view {:padding-bottom 150}
+        [preview/customizer state descriptor]
+        [rn/view
+         {:padding-vertical 60
+          :align-items      :center}
+         [quo/group-messaging-card (get-mock-data @state)]]]])))
+
+(defn preview-group-messaging-card
+  []
+  [rn/view
+   {:background-color (colors/theme-colors colors/white colors/neutral-90)
+    :flex             1}
+   [rn/flat-list
```

Comment:
I think we can move away from using the flat list approach in quo2 preview screens. @smohamedjavid wdyt? is it needed here? 🤔

Reply:
It depends on the number of `descriptors`(or options/props) of the component. If it has plenty of `descriptor` like `button` component, this will require a scroll view (we have been using a Flat list for some reason which I don't know) to ensure the content is scrollable in small screen devices too.  

If the component has no descriptor or one/two descriptors, I'm sure it will fit on the screen along with the component below. It all depends on the number of descriptors and the height of the component.